etc/alternc/templates/proftpd/conf.d/sftp.conf

```diff
@@ -0,0 +1,44 @@
+<VirtualHost 0.0.0.0>
+    SFTPEngine on
+    Port 2222
+    SFTPLog /var/log/proftpd/sftp.log
+
+    SFTPOptions AllowInsecureLogin
```

Comment:
AllowInsecureLogin seems to be a value for development environments. I don't think it's a good idea to run it in production. Is there are reason for including this?

@see www.proftpd.org/docs/contrib/mod_sftp.html#SFTPOptions

Reply:
to complete : we need accept password connection , otherwise profted accept only publickey connection